### PR TITLE
Deprecate FastUUID

### PR DESCRIPTION
**Pull request by stevenschlansker**
No longer needed with jdk9, fix is integrated upstream: https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8006627

### CODE REVIEW

**Comment by stevenschlansker** [comment]
TFW you can finally close down a personal bug from 2013!

File: pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>23.0</version>
+      <version>23.5-jre</version>
```
Comment (pennello) [inline pom.xml]: Can't we omit this version specification now that we're depending on parent 154?

Comment (pennello) [reply pom.xml]: Actually, should we still be depending on `otj-parernt-oss` here?  Shouldn't we be using the "Spring" parent?

Comment (stevenschlansker) [reply pom.xml]: I do agree we could do that, but let's defer re-parenting this to a separate change.

**Pull request merged**